src/components/form/form.style.js

```diff
@@ -51,6 +51,23 @@ export const StyledFormFooter = styled.div`
       bottom: 0;
     `}
   `}
+
+  ${({ fullWidthButtons }) => css`
```

Comment:
It might be better to add `fullWidthButtons` to the code block that destructures the `stickyFooter` prop as some of most of the styles are the same and could potentially be set twice if both are true etc.

I think the only difference is that `flex-direction: column; align-items: stretch;` are set when `fullWidthButtons` is true

Reply:
Yeah good point. Got so much tunnel vision trying to get the styles right, I missed that. Cheers.